### PR TITLE
Add avatarURL property to user model

### PR DESCRIPTION
**Pull request by BohdanOne**

### CODE REVIEW

File: Todo.API/src/models/user.js [hunk 1]
```diff
@@ -22,12 +22,16 @@ const userSchema = new mongoose.Schema({
     },
     tasks: {
         type: [ { type: mongoose.Schema.Types.ObjectId, ref: 'Task' } ],
+    },
+    avatarURL: {
+        type: String,
+        default: 'https://cdn.pixabay.com/photo/2015/03/04/22/35/head-659652_960_720.png'
```
Comment (DominikKrajcer) [inline Todo.API/src/models/user.js]: Magic numbers...

Comment (BohdanOne) [reply Todo.API/src/models/user.js]: yes, i know. this one is to be replaced by file we have already in src/images, i just wanted to have something there already

Comment (BohdanOne) [reply Todo.API/src/models/user.js]: user controllers already updated

Comment (BohdanOne) [reply Todo.API/src/models/user.js]: Ok, now i have a problem. So the model is in Todo.API on the backend, end the img to which i want to point as default value of avatrURL is in Todo.Client which is on the frontend. How should I make proper connection? Or should i move the image to the backend part of code, and the request it from the front (Anyway, link will be saved to db, so I'm not sure if all of that will work properly.. Now i have an idea how to solve this: delete default value in the model, and then in frontend - after fetching user data - make check if there is any value in avatarURL, if so use that value (should be valid url) as image src, if value is '' then use path to image in src/images - is it good solution? @DominikKrajcer

Comment (DominikKrajcer) [reply Todo.API/src/models/user.js]: Yes, that is good solution, a one that we apply in real applications. In current approach you would save redundant img url in the db, and there is probability that a server from which that img url derives would be not available or img might be deleted. We don't want to load our server with unnecessary data which can be stored on a client side. Btw, if you want to store an avatar in user objects you would be force to send image as... Just think about how you can send an image from client to server. Remember, if you won't figure it out I can help.

Comment (BohdanOne) [reply Todo.API/src/models/user.js]: I decided to use url's here as a quick solution to omit uploading images (but with plan to eventually implement that in future, i'm just afraid we don't have enough time now - i'm available only today, and i need this day to finish react course :/ ) Anyway, Olek already started to work on fixing this avatar stuff

**Comment by alek67890** [comment]
Current plan is to store avatar Url in db if user has custom avatar. 
In frond end normally will be set some default avatar and if in resposne from api is a avatarURL property then the avatar will be replaced using:

    if (res.data.avatarURL){
        let imgURL = `url(${tasksController.data.avatarURL})`
        document.querySelector(".user-avatar").style.backgroundImage = imgURL;
    }

just waiting for user login functionality.

**Pull request merged**